src/main.ts

```diff
@@ -6,7 +6,11 @@ import * as cookieParser from 'cookie-parser';
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   app.use(cookieParser());
-  app.useGlobalPipes(new ValidationPipe());
+  app.useGlobalPipes(
+    new ValidationPipe({
+      transform: true,
+    }),
```

Comment:
```suggestion
    new ValidationPipe({
      transform: true,
      whitelist: true,
    }),
```
Thêm whitelist để chỉ nhận những filed có trong request validation https://docs.nestjs.com/techniques/validation